index.bs

```diff
@@ -7984,20 +7984,19 @@ JavaScript Array values.
     given an iterable |iterable| and an iterator getter |method|,
     perform the following steps:
 
-    1.  Let |iter| be [=?=] <a abstract-op>GetIterator</a>(|iterable|, <emu-const>sync</emu-const>, |method|).
+    1.  Let |iteratorRecord| be [=?=] <a abstract-op>GetIteratorFromMethod</a>(|iterable|, |method|).
     1.  Initialize |i| to be 0.
     1.  Repeat
-        1.  Let |next| be [=?=] <a abstract-op>IteratorStep</a>(|iter|).
-        1.  If |next| is <emu-val>false</emu-val>,
+        1.  Let |next| be [=?=] <a abstract-op>IteratorStepValue</a>(|iteratorRecord|).
+        1.  If |next| is <emu-const>done</emu-const>,
```

Comment:
Should this be uppercase DONE as is done in JS?

Reply:
No, that's a visual choice they apply via CSS in the default ecmarkup stylesheet:

https://github.com/tc39/ecmarkup/blob/c0c4f82924077eea14d95f1eb6bd067239749554/css/elements.css#L252-L258

![image](https://github.com/whatwg/webidl/assets/19366641/2fcb3a0f-4b91-412a-a093-2d8d3dcebf57)

Reply:
Ooh weird. Thanks!